data/partOrigin.rdf

```diff
@@ -1,399 +1,411 @@
+    <skos:prefLabel xml:lang="en">Quail</skos:prefLabel>
+  </rdf:Description>
+
+  <rdf:Description rdf:about="http://static.datafoodconsortium.org/data/partOrigin.rdf#Roster">
+    <rdf:type rdf:resource="skos:Concept"/>
+    <skos:broader rdf:resource="http://static.datafoodconsortium.org/data/partOrigin.rdf#AnimalPartOrigin"/>
+    <skos:inScheme rdf:resource="http://static.datafoodconsortium.org/ontologies/DFC_ProductGlossary.owl#PartOrigin"/>
+    <skos:narrower>
+      <rdf:Description rdf:about="http://static.datafoodconsortium.org/data/partOrigin.rdf#RosterBody">
+        <rdf:type rdf:resource="skos:Concept"/>
+        <skos:broader rdf:resource="http://static.datafoodconsortium.org/data/partOrigin.rdf#Roster"/>
+        <skos:inScheme rdf:resource="http://static.datafoodconsortium.org/ontologies/DFC_ProductGlossary.owl#PartOrigin"/>
+        <skos:prefLabel xml:lang="fr">Partie de coq</skos:prefLabel>
+        <skos:prefLabel xml:lang="en">Rooster's body</skos:prefLabel>
+      </rdf:Description>
+    </skos:narrower>
+
+    <skos:prefLabel xml:lang="fr">Coq</skos:prefLabel>
+    <skos:prefLabel xml:lang="en">Rooster</skos:prefLabel>
+  </rdf:Description>
+
+  <rdf:Description rdf:about="http://static.datafoodconsortium.org/data/partOrigin.rdf#Sheep">
+    <rdf:type rdf:resource="skos:Concept"/>
+    <skos:broader rdf:resource="http://static.datafoodconsortium.org/data/partOrigin.rdf#AnimalPartOrigin"/>
+    <skos:inScheme rdf:resource="http://static.datafoodconsortium.org/ontologies/DFC_ProductGlossary.owl#PartOrigin"/>
+    <skos:narrower>
+      <rdf:Description rdf:about="http://static.datafoodconsortium.org/data/partOrigin.rdf#SheepBody">
+        <rdf:type rdf:resource="skos:Concept"/>
+        <skos:broader rdf:resource="http://static.datafoodconsortium.org/data/partOrigin.rdf#Sheep"/>
+        <skos:inScheme rdf:resource="http://static.datafoodconsortium.org/ontologies/DFC_ProductGlossary.owl#PartOrigin"/>
+        <skos:prefLabel xml:lang="fr">Partie de mouton</skos:prefLabel>
+        <skos:prefLabel xml:lang="en">Sheep's body</skos:prefLabel>
+      </rdf:Description>
+    </skos:narrower>
+
+    <skos:prefLabel xml:lang="fr">Mouton</skos:prefLabel>
+    <skos:prefLabel xml:lang="en">Sheep</skos:prefLabel>
+  </rdf:Description>
+
+  <rdf:Description rdf:about="http://static.datafoodconsortium.org/data/partOrigin.rdf#UnknownPartOrigin">
+    <rdf:type rdf:resource="skos:Concept"/>
+    <skos:inScheme rdf:resource="http://static.datafoodconsortium.org/ontologies/DFC_ProductGlossary.owl#PartOrigin"/>
+    <skos:prefLabel xml:lang="fr">Partie d'origine inconnue</skos:prefLabel>
+    <skos:prefLabel xml:lang="en">Unknown part origin</skos:prefLabel>
+    <skos:topConceptOf rdf:resource="http://static.datafoodconsortium.org/ontologies/DFC_ProductGlossary.owl#PartOrigin"/>
+  </rdf:Description>
+
+  <rdf:Description rdf:about="http://static.datafoodconsortium.org/ontologies/DFC_ProductGlossary.owl#PartOrigin">
```

Comment:
I've moved this to the top & adjusted the formatting to better align with the previous structure. The RDF generator sorts data items into alphabetical order. This makes sense to me and I propose we do that in the future for all files (will hopefully make diffs easier in the future).

I've also amended the structure so that all the narrower individuals are nested within their broader categories.